docs/manuals/core/entities/scheduling-and-job-execution.md

```diff
@@ -1,26 +1,50 @@
-# Submit a scenario, pipeline or task.
+# Submit a scenario, pipeline, or task
+
+In a Taipy application, running the Core service is required to execute jobs.
```

Comment:
Whao. That is really a constraint, for a minor version.
So no legacy app that uses only Core will work any longer?

Reply:
It is a 1 line of code fix to run legacy app with 2.1.
What do you think?